lib/Echidna/Test.hs

```diff
@@ -15,18 +15,20 @@ import Data.Foldable (traverse_)
 import Data.Has (Has(..))
 import Data.Maybe (fromMaybe)
 import Data.Text (Text)
-import EVM (Error(..), VMResult(..), VM, calldata, result, state)
+import EVM
 import EVM.ABI (AbiValue(..), abiCalldata, encodeAbiValue)
 import EVM.Types (Addr)
 
 import qualified Data.ByteString as BS
+import qualified Data.Text as T
 
 import Echidna.ABI
 import Echidna.Exec
 import Echidna.Solidity
 import Echidna.Transaction
 import Echidna.Types.Buffer (viewBuffer)
 import Echidna.Types.Tx (TxCall(..), Tx(..), TxConf, propGas, src)
+import Echidna.Events   (EventMap, extractEvents)
```

Comment:
you don't have to align the qualified imports

Reply:
Fixed.